prelude/slang-cuda-prelude.h

```diff
@@ -1615,26 +1615,63 @@ struct UniformState;
 
 // ---------------------- OptiX Ray Payload --------------------------------------
 #ifdef SLANG_CUDA_ENABLE_OPTIX
+struct RayDesc
+{
+    float3 Origin;
+    float  TMin;
+    float3 Direction;
+    float  TMax;
+};
+
 static __forceinline__ __device__
 void *unpackOptiXRayPayloadPointer(uint32_t i0, uint32_t i1)
 {
-	const uint64_t uptr = static_cast<uint64_t>(i0) << 32 | i1;
-	void*           ptr = reinterpret_cast<void*>(uptr);
-	return ptr;
+    const uint64_t uptr = static_cast<uint64_t>(i0) << 32 | i1;
+    void*           ptr = reinterpret_cast<void*>(uptr);
+    return ptr;
 }
```

Comment:
Note, I converted these tabs to spaces to stay consistent with the rest of the prelude file